showItemPageBE/src/test/java/com/talentpath/shamazin/showItemPage/services/ItemServiceTest.java

```diff
@@ -24,33 +21,32 @@
 
     @Autowired
     ItemService itemServe;
-    @Autowired
```

Comment:
We should keep these for testing, unless you are planning to separate them all into separate files.